File: springdoc-openapi-ui/src/test/java/test/org/springdoc/ui/app11/ExampleController.java
package test.org.springdoc.ui.app11;

import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.PostMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class ExampleController {

    @PostMapping("/post")
    public ResponseEntity<String> postExample() {
        return ResponseEntity.ok("Successful post");
    }

    @GetMapping("/get")
    public ResponseEntity<String> getExample() {
        return ResponseEntity.ok("Successful get");
    }
}

File: springdoc-openapi-tests/springdoc-openapi-kotlin-webflux-tests/src/test/kotlin/test/org/springdoc/api/v30/app10/ExampleController.kt
package test.org.springdoc.api.v30.app10

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestParam
import org.springframework.web.bind.annotation.RestController
import test.org.springdoc.api.v30.app8.Greeting

data class Greeting(val greeting: String)

@RestController
class ExampleController {
	@GetMapping("/")
	fun greet(@RequestParam name: String?) = Greeting("Hello ${name ?: "world"}")

	@GetMapping("/test")
	fun test(@RequestParam name: String) = Greeting("Hello $name")

	@GetMapping("/test-with-default")
	fun testWithDefault(@RequestParam(defaultValue = "world") name: String) = Greeting("Hello $name")
}